analysis_options_plus.yaml

```
@@ -1,17 +1,14 @@
 # This file was copied from plus_plugins/analysis_options.yaml.
 
-include: package:pedantic/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
 
 analyzer:
-  enable-experiment:
-    - non-nullable
   exclude:
     # Ignore generated files
     - '**/*.g.dart'
     - 'lib/src/generated/*.dart'
 
 linter:
   rules:
-    public_member_api_docs: true
     prefer_final_in_for_each: true
-    prefer_final_locals: true
\ No newline at end of file
+    prefer_final_locals: true
```

```suggestion
    prefer_final_locals
```
https://github.com/fluttercommunity/plus_plugins/blob/main/analysis_options.yaml

---

Done!